src/suites/positive/validation/response-validator.ts

```diff
@@ -30,6 +31,23 @@ class ResponseValidator extends PositiveValidator {
         return;
       }
 
+      // if the Accept header is populated in the request
+      const acceptHeader = this.operation.parameters?.filter(
+        (parameter) => parameter.in === 'header' && parameter.name === 'Accept',
+      );
+      if (acceptHeader && acceptHeader.length > 0) {
+        // then check that the media type of the response is compatible with what was specified
+        const responseType = new MediumType(contentType);
+        const acceptType = new MediumType(acceptHeader[0].example);
+        if (!responseType.match(acceptType)) {
+          this.addMessage(
+            Type.MediaTypeMismatch,
+            [],
+            [acceptType, responseType],
+          );
```

Comment:
I wonder if this check doesn't pass if we should also return, like we do above. If these content types don't match I'm not sure if it makes sense to continue validation?

Reply:
Is there any downside to continuing with validation?   I was thinking as long as the response is parse-able and a there's a schema for it, it couldn't hurt to proceed.  After all, a media type mismatch could be something wrong with just the accept header, like a typo, which wouldn't affect how the response is validated.

Reply:
Ok, that works for me!